spec/examples/validations/css.spec.js

```diff
@@ -73,6 +73,15 @@ describe('css', function() {
     )
   );
 
+  it('gives a useful error when there is no closing curly brace', () =>
+    assertFailsValidationWith(
+      css,
+      `p {
+        display: block;`,
+      'missing-closing-curly'
+    )
+  );
+
```

Comment:
I first thought the the missing semicolon test between lines 109-124 already validates for this error. After looking into it again, PrettyCSS seems to only fail this test as long as the missing semicolon isn't placed on the last property declaration.

How should I approach this? Should I simply add another `it()` test or add on to the `context('missing semicolon') piece?